lib/services/sitemaps/index.js

```diff
@@ -123,7 +123,20 @@ function streamNewsEntries(site) {
       size: '50',
       body: {
         sort: [{lastmod: 'desc'}],
-        query: {bool: {filter: {term: {site}}}}
+        query: {
+          bool: {
+            filter: [{
+              term: {site}
+            }, {
+              range: {
+                publication_date: {
+                  gte : 'now-2d/d',
```

Comment:
how strict is google on 'two days'? `/d` rounds down to the nearest day, so a query that's executed on 23:59 on Wednesday would grab articles from the past ~72 hours. should the query instead be for `now-48h`?

Reply:
My thought is that if Google meant "the last 48 hours" the guide would more precisely say "the last 48 hours." But it's ambiguous. I'll switch to the 48-hour window b/c that's what the Washington Post appears to be using for their news sitemaps.